Acrolinx quality improvements: auditpol-remove.md (#6942)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/administration/windows-commands/auditpol-remove.md

---
title: auditpol remove
description: Reference article for the auditpol remove command; auditpol remove removes the per-user audit policy for a specified account or all accounts.
ms.topic: reference
author: JasonGerend
ms.author: jgerend
ms.date: 01/06/2023
---
# auditpol remove

Applies to: Windows Server 2022, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012

The auditpol remove command removes the per-user audit policy for a specified account or all accounts.

> To perform *remove* operations on the *per-user* policy, you must have **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *remove* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that isn't necessary to perform the overall *remove* operations.

Reviewer: ```suggestion
To perform *remove* operations on the *per-user* policy, you must have **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *remove* operations if you've the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows more access than is necessary to perform the overall *remove* operations.
```
